portfolio/src/main/webapp/index.html

```diff
@@ -6,18 +6,14 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComments()">
```

Comment:
I don't think you only need the reference, try without the parentheses.    <body onload="getComments">

Reply:
Actually I might be wrong